app/views/users/liked.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div class="col md-6 offset-3">
+    <h1>
+      <%= @user.username %>'s Liked posts
+    </h1>
+
+    <ul class="nav nav-pills">
+      <li class="nav-item">
+        <%= link_to "Posts", user_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Liked photos", liked_photos_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Feed</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Followers</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Following</a>
+      </li>
```

Comment:
You missed updated the links on the liked tab, but they work everywhere else.  Consider using a partial here instead that you can just reference on every page.

Reply:
Hi Jared! Thank you for taking a look; I have updated my code to reflect your suggestions :)